common/test/acceptance/pages/studio/users.py

```diff
@@ -254,6 +254,7 @@ def click_delete(self):
         self.wait_for_element_visibility('.prompt', 'Prompt is visible')
         self.wait_for_element_visibility('.prompt .action-primary', 'Confirmation button is visible')
         self.q(css='.prompt .action-primary').click()
+        self.wait_for_element_absence('.page-prompt .is-shown', 'Confirmation prompt is hidden')
         wait_for_ajax_or_reload(self.browser)
```

Comment:
I'll try removing it and running the test another 30 times. It's great how easy it is now.